metadata/src/main/java/org/apache/kafka/metadata/authorizer/StandardAuthorizer.java

```diff
@@ -207,4 +216,45 @@ static AuthorizationResult getDefaultResult(Map<String, ?> configs) {
         if (configValue == null) return DENIED;
         return Boolean.parseBoolean(configValue.toString().trim()) ? ALLOWED : DENIED;
     }
+
+    @Override
+    public void withPluginMetrics(PluginMetrics metrics) {
+        this.authorizerMetrics = new AuthorizerMetrics(metrics);
+    }
+
+    private class AuthorizerMetrics {
```

Comment:
Have you considered exposing `AuthorizerMetrics` as public APIs for custom Authorizer?

Reply:
Frankly, no I didn't. This would require updating KIP-877 if we wanted to do so.